pynautobot/core/endpoint.py

```diff
@@ -394,6 +394,56 @@ def count(self, *args, api_version=None, **kwargs):
 
         return ret.get_count()
 
+    def run(self, *args, api_version=None, **kwargs):
```

Comment:
I don't think this should be part of the endpoint class, but should be specific to a job class implementation

Reply:
@jeffkala